src/Chapter1.hs

```diff
@@ -640,8 +656,8 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = if div n 10 == 0 then n else firstDigit (div n 10)
```

Comment:
I see, that for a negative number like -9, this implementation will return the negative number itself instead of the last digit. 

But it works nicely on other cases 👍🏼